recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - {{ compiler('c') }}
     - setuptools >=45
     - setuptools-scm >=4
+    - setuptools_scm_git_archive
```

Comment:
We would have to add this in the main repository (before the archives are built) if we want to go that route.

Reply:
I don't think we need that.
I switched to using the archives in PyPi that already contain the `__version`